x-pack/plugins/reporting/export_types/common/execute_job/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { addForceNowQuerystring } from './add_force_now_query_string';
+export { decryptJobHeaders } from './decrypt_job_headers';
+export { getConditionalHeaders } from './get_conditional_headers';
+export { getCustomLogo } from './get_custom_logo';
+export { omitBlacklistedHeaders } from './omit_blacklisted_headers';
+export { getAbsoluteUrlFactory } from './get_absolute_url';
+export { createMockServer } from '../../../test_helpers/create_mock_server';
```

Comment:
I don't think we want to mix code for a test utility in this module

Reply:
Agreed, I will remove this line and will include a direct reference to ../../../test_helpers/create_mock_server in the test files. Or do you think an index,test,ts is better?

Reply:
I'd go with adding `index.test.js` so that we don't have 6 files all reaching a far way up to import the module.